pkg/pipeline/extract/aggregate/aggregate.go

```diff
@@ -213,3 +229,21 @@ func (aggregate Aggregate) GetMetrics() []config.GenericMap {
 
 	return metrics
 }
+
+func (aggregate Aggregate) cleanupExpiredEntries() {
+	nowInSecs := time.Now().Unix()
+	expireTime := nowInSecs - expiryTime
```

Comment:
Where is `expiryTime` defined?
The same `expiryTime` (default value) for all aggregates? Not configurable? OK

Reply:
Currently --- yes one default - I added code to have different expirations for each aggregate but for now this is common to all. Also, currently, my thinking is that this is a big hardcoded value (10 minutes). When we will start to benchmark, if needed we will add that to the API and split to fit the needs of each aggregate. At this point, I do not see a good reason to differentiate between the current aggregates that we have in the default FLP confiuration